Runtimes/Core/SwiftOnoneSupport/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+add_library(swiftSwiftOnoneSupport
+  SwiftOnoneSupport.swift
+  "${PROJECT_SOURCE_DIR}/linker-support/magic-symbols-for-install-name.c")
+
+set_target_properties(swiftSwiftOnoneSupport PROPERTIES Swift_MODULE_NAME SwiftOnoneSupport)
+
+if(APPLE AND BUILD_SHARED_LIBS)
+  target_link_options(swiftSwiftOnoneSupport PRIVATE "SHELL:-Xlinker -headerpad_max_install_names")
+endif()
+
+target_compile_options(swiftSwiftOnoneSupport PRIVATE
+  $<$<COMPILE_LANGUAGE:Swift>:-parse-stdlib>
+  "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-Xllvm -sil-inline-generics=false>"
+  "$<$<COMPILE_LANGUAGE:Swift>:SHELL:-Xfrontend -validate-tbd-against-ir=none>"
```

Comment:
Can you file an issue to track this? This seems like a bug.

Reply:
Looking at https://github.com/swiftlang/swift/pull/16838, the use of this flag is on purpose -- I noticed that the comment that accompanied the addition of the flag got lost in recent edits, so I reinstated it.